### DISCUSSION TITLE
Getting "The current request does not have a file in a key" in Laravel Jetstream

### DISCUSSION THREAD

**Discussion by denzeldenzeldenzel** [spatie/laravel-medialibrary#2166] [Q&A] [answered]
I'm trying to use this library with the built-in profile picture upload in laravel jetstream. Right now I'm just trying to make this piece of code work: 

```

 if (isset($input['photo'])) {
          
            $user->addMediaFromRequest('photo')->toMediaCollection('photos');
        }

```

and i'm getting this as an error:
"Spatie\MediaLibrary\MediaCollections\Exceptions\RequestDoesNotHaveFile
The current request does not have a file in a key named photo" 

I'm not sure what i'm doing wrong, Thanks in advance.

**Comment by freekmurze** [comment]
What do you get when you dd $request->all()?

**Reply by denzeldenzeldenzel** [reply]
Hi, I get Error
Call to a member function all() on array when i do dd($input->all());

**Comment by denzeldenzeldenzel** [comment, marked as answer]
I just realized that the implementation isn't using laravel requests.
